test/config-luatest/helpers.lua

```diff
@@ -226,6 +226,7 @@ local function reload_success_case(g, opts)
     local script_2 = opts.script_2
     local options = assert(opts.options)
     local verify_2 = assert(opts.verify_2)
+    local verify_args_2 = opts.verify_args_2
```

Comment:
Nit: While we're here it would be good to add `verify_args` and `verify_args_2` into the list of options in the comment for the function:

```diff
 -- * opts.roles
 -- * opts.script
 -- * opts.options
 -- * opts.verify
+-- * opts.verify_args
 --
 --   Same as in success_case().
@@
 -- * opts.verify_2
 --
 -- Verify test invariants after config:reload().
+--
+-- * opts.verify_args_2
+--
+-- Arguments for the verify_2 function.
```

Reply:
Applied.